### PR TITLE
treewide: switch to zoogie's release download, remove static version

### PR DESCRIPTION
**Pull request by lifehackerhansol**

### CODE REVIEW

File: _pages/en_US/troubleshooting.txt [hunk 1]
```diff
@@ -277,7 +277,7 @@ Join [Nintendo Homebrew on Discord](https://discord.gg/MWxPgEp) and explain what
...
+If your files are in the correct locations, re-create the save using the [PicHaxx Save Tool](https://3dstools.nhnarwhal.com/#/pichaxx), then place it in `Nintendo 3DS` -> `<ID0>` -> `<ID1>` -> `title` -> `00040000` -> `0017c100` -> `data`. Ensure that the file is named exactly `00000001.sav` and that you used your console's `movable.sed` to create it. Re-download [otherapp](https://github.com/zoogie/ninjhax2.x/releases/download/v11.17/otherapps_with_CfgS_11.17.zip), place the `.bin` file relevant to your console to the root of your SD card, and rename it to `otherapp.bin`. Do not add the `.bin` extension if you do not already see it.
```
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:280]: "Re-download otherapp" is kinda weird, at least add an article

File: _pages/en_US/troubleshooting.txt [hunk 2]
```diff
@@ -291,7 +291,7 @@ Your console already has custom firmware. You should [check for CFW](checking-fo
 {% capture compat %}
 <summary><u>"An error has occurred, forcing the software to close..." (white message box)</u></summary>
 
-There is an issue with your `otherapp.bin` file. Download [otherapps.zip](/assets/otherapps.zip), place the `.bin` file relevant to your console to the root of your SD card, and rename it to `otherapp.bin`. Do not add the `.bin` extension if you do not already see it.
+There is an issue with your `otherapp.bin` file. Download [otherapp](https://github.com/zoogie/ninjhax2.x/releases/download/v11.17/otherapps_with_CfgS_11.17.zip), place the `.bin` file relevant to your console to the root of your SD card, and rename it to `otherapp.bin`. Do not add the `.bin` extension if you do not already see it.
```
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:294]: ditto

**Review by lilyuwuu — COMMENTED** [review]
maybe replace just 'otherapp' with 'HBL otherapps' or something

File: _pages/en_US/troubleshooting.txt (hunk 1 above)
Comment (lifehackerhansol) [reply _pages/en_US/troubleshooting.txt]: updated, is that better?

**Review by lilyuwuu — APPROVED** [review]
ya sure that works

**Pull request merged**